source/index.html.md.erb

```diff
@@ -388,6 +388,63 @@ This core hierarchy of Sheet > Column > Row > Cell is essential to working with
 
 # Sample Code and SDKs
 
+> Install the SDK
+
+```javascript
+npm install smartsheet
+```
+
+```csharp
+Install-Package smartsheet-csharp-sdk
+```
+
+```java
+git clone https://github.com/smartsheet-platform/smartsheet-java-sdk.git
+cd smartsheet-java-sdk
+mvn package
+```
+
+```python
+$ pip install smartsheet-python-sdk
+```
+
+> Load the client
+
+```javascript
+// Initialize client SDK
+var smartsheet = client.createClient({ accessToken: token });
```

Comment:
Don't you want to use the sample token here?

Reply:
Done. (I was torn between how the token is set up in a config file vs showing it. But since this is a sample, it seems more clear to show it.)